### PR TITLE
Feat/database migration

### PR DESCRIPTION
**Pull request by Wordz278**
initial pull request

### CODE REVIEW

**Review by sinethembazulu045 — APPROVED** [review]

File: app/development.sql [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+Julia	Cage	JCage92	9412075657950	JCage@gmail.com	C27_Web_Dev	JCage92
+Timu	Grey	TimuGrey009!	9612075607901	TGrey@gmail.com	C27_Web_Dev	TimuGrey009!
+Fortunate	Blue	MoneyFloyd	9212075607902	FBlue1000@gmail.com	C27_Web_Dev	FBlue1000
+Micky	Black	MickyB009!	9412075607903	MK@gmail.com	C27_Web_Dev	MK009!
+Zoe	Yellow	ZoeYel09	9212075607704	ZY@gmail.com	C27_Web_Dev	ZoeY
+Jake	Rod	JRod92	9412075657951	JRod@gmail.com	C28_Data_Scientists	JRod92
+Charly	Pop	CPop009!	9512075607909	CPop@gmail.com	C28_Data_Scientists	CPop009!
+Allan	Brown	ABrown	9212075607903	ABrown1000@gmail.com	C28_Data_Scientists	ABrown1000
+Sam	Husk	SamH009!	9312073687909	SamH@gmail.com	C28_Data_Scientists	SamH009!
+Joe	Campbell	JoeCl09	9512075607705	JoeC@gmail.com	C28_Data_Scientists	JoeC
```
Comment (JanTeffo442) [inline app/development.sql:95]: Nice work with the sql dump which shows the database tables and content.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+# Executing the project:  
+1. Source the envirnoment variables using the terminal:  
+    source env.sh
+2. cd into the project and cd into the composition folder to start up postgresql using the docker-compose script:  
+    docker-compose up or for installing docker-compose see https://docs.docker.com/compose/install/  
+3. run the setup script to enable project modules importing:  
+    python setup.py develop
+4. cd into the app directory and create tables of the development database:
+    from app import db  
+    db.create_all()
```
Comment (JanTeffo442) [inline README.md]: README file is clear and illustrative.

File: app/create_c26_recruits.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    cohort = db.Column(db.String(100))
+
+    def __init__(self, first_name, surname, chatname, github_name, id_number, personal_email_address, cohort):
+        self.first_name = first_name
+        self.surname = surname
+        self.chatname = chatname
+        self.github_name = github_name
+        self.id_number = id_number
+        self.personal_email_address = personal_email_address
+        self.cohort = cohort
```
Comment (JanTeffo442) [inline app/create_c26_recruits.py]: Code is clean and awesome use of function to add data into your database table.

**Review by JanTeffo442 — APPROVED** [review]
Well done.

**Pull request merged**